recipe/meta.yaml

```diff
@@ -64,62 +67,44 @@ outputs:
         - fdtget --help
         - fdtoverlay --help
         - fdtput --help
-        # There is only one compilation unit needed from the toplevel project. Running the tests
-        # from inside tests subdir removes the binaries and library target from the prereqs and
-        # that's what we want for testing the packaged artifacts. So, we copy three tiny files
-        # from the toplevel project into tests and then we can 'cleanly' rerun the tests using the
-        # packaged artifacts
-        - cp util.c util.h version_gen.h tests
-        # use Makefile.tests directly so that binaries won't be built but used from installed package
-        # NO_PYTHON because we test that in pylibfdt
-        # VECHO=: to enable Makefile.tests to stand on it's own
-        # LDLIBS=-lfdt to correctly find installed libfdt instead of having in-tree library an explicit dependency listed in the link command
-        # LIBDL= needs to also include -lfdt because of how it is used in rules to override the  builtin and not use LDLIBS
-        # TESTS_PREFIX=./ to work around 'cd $TEST_PREFIX' in check recipe
-        # TEST_BINDIR="$PREFIX/bin" because testutils.sh hardcodes the paths to binaries, defaulting to ../
-        - 'NO_PYTHON=1 TEST_BINDIR="$PREFIX/bin" make -C tests -f Makefile.tests VECHO=: LDLIBS=-lfdt tests_clean check TESTS_PREFIX=./ LIBDL="-lfdt -ldl"'  # [linux]
-        # make check doesn't build cleanly on osx due to GCC-specific directive usage in tests/trees.S, so we don't do that testing on osx
     about:
       summary: 'Linux Kernel Device Tree Compiler (dtc)'
       description: |
        Device Tree Compiler (dtc) toolchain for working with device
        tree source and binary files
       license: GPL-2.0-or-later
       license_family: GPL2
-      license_file: 
+      license_file:
         - GPL
         - README.license
   - name: pylibfdt
-    script: install.sh
+    build:
+      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
     requirements:
       build:
         - {{ compiler('c') }}
-        - make
-        - swig
+        - {{ stdlib('c') }}
+        - swig <4.2
```

Comment:
Can you comment why the cap on swig is necessary?

Reply:
I was hitting some bug with swig that I fixed by switching to 4.2, but it builds with swig 4.3 now. Removed the limit.